ocaml/dune

```diff
@@ -60,6 +72,7 @@
    ; remove -w -67 by adding the camlinternalMenhirLib hack like the Makefile
  ))
  (ocamlopt_flags (:include %{project_root}/ocamlopt_flags.sexp))
+ (preprocessor_deps parser_types.ml parser_types.mli)
```

Comment:
Is this line still needed?

Reply:
Oops, I didn't consider that removing the echo obsolesced that.

Reply:
Oops, I didn't consider that removing the echo obsolesced that.